kotlinx.fuzz.api/build.gradle.kts

```diff
@@ -1,10 +1,10 @@
 plugins {
-    id("org.plan.research.kotlinx-fuzz-module")
+    id("kotlinx.fuzz.src-module")
 }
 
 dependencies {
-    implementation("com.github.curious-odd-man:rgxgen:$RGX_GEN_VERSION")
-    implementation("org.junit.platform:junit-platform-engine:$JUNIT_PLATFORM_VERSION")
+    implementation(libs.rgxgen)
+    implementation(libs.junit.platform.engine)
```

Comment:
Please, add `testRuntimeOnly(libs.junit.jupiter)` here. For some reason, tests doesn't work on my machine withot this line (and it's very weird it works on CI)